packages/cli/src/modules/start/commands/package/start/startFrontend.ts

```diff
@@ -22,6 +22,7 @@ import {
 } from '../../../../build/lib/bundler';
 import { paths } from '../../../../../lib/paths';
 import { BackstagePackageJson } from '@backstage/cli-node';
+import { hasReactDomClient } from '../../../../build/lib/bundler/hasReactDomClient';
```

Comment:
Just wondering why not import this on line 21 or is it missing from the module export?

Reply:
We're likely to merge `build` and `start`, too much in common between those two to keep them apart